Content.Shared/GameObjects/Components/GUI/SharedStrippableComponent.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using Content.Shared.GameObjects.Components.Inventory;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Components.UserInterface;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.GameObjects.Components.GUI
+{
+    public class SharedStrippableComponent : Component
+    {
+        public override string Name => "Strippable";
+
+        [NetSerializable, Serializable]
+        public enum StrippingUiKey
+        {
+            Key,
+        }
+    }
+
+    [NetSerializable, Serializable]
+    public class StrippingInventoryButtonPressed : BoundUserInterfaceMessage
+    {
+        public EquipmentSlotDefines.Slots Slot { get; }
```

Comment:
You can import static members for EquipmentSlotDefines with `
using static Content.Shared.GameObjects.Components.Inventory.EquipmentSlotDefines;`, removing the need to specify it every time.

Reply:
done